### PR TITLE
DB-9386 Increase heap size for standalone 

### PR DESCRIPTION
**Pull request by dgomezferro**
Make sure we create ZK structure before trying to access it
Some other small memory usage reductions on metrics

### CODE REVIEW

**Comment by dgomezferro** [comment]
jenkins please test branch @cdh6.3.0

**Comment by spliceci** [comment]
TEST FAILED

**Comment by dgomezferro** [comment]
jenkins please test branch @cdh6.3.0

**Comment by spliceci** [comment]
TEST FAILED

**Comment by dgomezferro** [comment]
jenkins please test branch @cdh6.3.0

**Comment by spliceci** [comment]
TEST FAILED

File: platform_it/src/main/resources/hadoop-metrics2-hbase.properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (arnaud-splice) [inline platform_it/src/main/resources/hadoop-metrics2-hbase.properties:1]: What's the point of this file? It seems fully commented out.

Comment (dgomezferro) [reply platform_it/src/main/resources/hadoop-metrics2-hbase.properties:1]: This file overrides the default that has some of those lines uncommented. We could certainly have an empty file, I just left the original for reference/example if we ever need to enable some of them.

**Pull request merged**